src/consts.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
         trayEventRightClicked: "tray.event.right.clicked",
         webContentsEventLogin: "web.contents.event.login",
         webContentsEventLoginCallback: "web.contents.event.login.callback",
+        webContentsExecutedJavaScript: "web.contents.executed.javascript",
```

Comment:
It seems you forgot `window.` in front of both name and value of this constant. Indeed I wanted to name it `window.event.web.contents.executed.javascript` instead.